Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7
+
+RUN mkdir /src
```

Comment:
Hi @carltongibson.

A tiny comment about `RUN mkdir /src`. The `WORKDIR` instruction creates the directory if it doesn't exist.

Documentation [quote](https://docs.docker.com/engine/reference/builder/#workdir):
>  If the `WORKDIR` doesn’t exist, it will be created even if it’s not used in any subsequent `Dockerfile` instruction.